app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/tokens/PerilCommonTokens.kt

```diff
@@ -0,0 +1,48 @@
+package com.hedvig.android.design.system.hedvig.tokens
+
+import androidx.compose.animation.core.AnimationSpec
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.ui.unit.dp
+
+internal object PerilCommonTokens {
+  val IconAnimationSpec: AnimationSpec<Float> = spring(stiffness = Spring.StiffnessMediumLow)
+  val ContainerShape = ShapeKeyTokens.CornerLarge
+  val ContainerColor = ColorSchemeKeyTokens.SurfacePrimary
+  val TextColor = ColorSchemeKeyTokens.TextPrimary
+  val NumbersColor = ColorSchemeKeyTokens.TextTertiary
+  val DefaultIconColor = ColorSchemeKeyTokens.FillPrimary
+  val DisabledTextColor = ColorSchemeKeyTokens.TextDisabled
+  val DisabledIconColor = ColorSchemeKeyTokens.FillDisabled
+  val PlusIconSize = 24.dp
+}
+
+internal object PerilSmallTokens {
+  val CircleSize = 16.dp
```

Comment:
Mb I'm reading the figma wrong? it's 24 in the Large size and 16 in Small. 

Well, tbh it's a bit unclear here. For Large size, it's
```
Icon/24/Circle-Filled
Icon · 24 x 24 (px)
```
but for Small, it's
```
Icon/16/Circle-Filled
Icon · 20 x 20 (px)
```

Reply:
Ah, the name has 16 but the icon itself is 20. Hence the confusion then?
I suppose let's just go with 20 since that's what the design visually shows then?